copilot/core/routes.py

```diff
@@ -446,3 +456,45 @@ def attach_file(file: UploadFile = File(...)):
     with temp_file_path.open("wb") as buffer:
         shutil.copyfileobj(file.file, buffer)
     return {"answer": str(temp_file_path)}
+
+@traceable
+@core_router.post("/checkCopilotHost")
+def check_copilot_host(authorization: str = Header(None)):
+    try:
+        etendo_host_docker = etendo_utils.get_etendo_host()
+
+        if not authorization or not authorization.startswith("Bearer "):
+            raise HTTPException(status_code=401, detail="Authorization token is missing or invalid")
+
+        token = authorization.split("Bearer ")[1]
```

Comment:
you are getting the bearer token, but you are not using it... the Endpoint works?